### ISSUE TITLE
character string is not in a standard unambiguous format, AL162001 prblty

### ISSUE THREAD

**Issue by timtrice** [ropensci/rrricanes#59]
[AL162001 Adv 1](http://www.nhc.noaa.gov/archive/2001/prb/al162001.prblty.001.html) lists time as "NOON'. 

Invalid

```
ZCZC MIASPFAT1 ALL
TTAA00 KNHC DDHHMM
HURRICANE NOEL SPECIAL PROBABILITIES NUMBER  1
NATIONAL WEATHER SERVICE MIAMI FL
NOON AST MON NOV 05 2001
```

Valid:

```
ZCZC MIASPFAT1 ALL
TTAA00 KNHC DDHHMM
HURRICANE NOEL PROBABILITIES NUMBER   2
NATIONAL WEATHER SERVICE MIAMI FL
5 PM AST MON NOV 05 2001
```

So "NOON" should be "12 PM".

**Comment by timtrice** [comment]
I've added a conditional that searches for the specific text within the header:

```
NOON AST MON NOV 05 200
```

and replaces with

```
12 PM AST MON NOV 05 200
```

I'm not 100% sure this is the best method. Originally I intended on modifying the pattern. However, the absence of the AM/PM indicator would throw off the str_match function (so instead of having *n* columns that matched, we would have *n -1* which would mean adding more code). 

If this does become a prevalent issue then I will look to modify it. 

Resolved in commit c1dcbec.

**Comment by timtrice** [comment]
This instance occurs again in [AL212003, prblty #2](http://www.nhc.noaa.gov/archive/2003/prb/al212003.prblty.002.shtml?).  Have modified the check to look for a specific pattern, 

```r
    if (stringr::str_count(header, pattern = "\nNOON [:upper:]{3} [:upper:]{3} [:upper:]{3} [:digit:]{2} [:digit:]{4}\n"))
        header <- stringr::str_replace(header, "\n(NOON)( [:upper:]{3} [:upper:]{3} [:upper:]{3} [:digit:]{2} [:digit:]{4})\n", "\n12 PM\\2\n")
```

This change is reflected in commit 9decc9a.
